### ISSUE TITLE
iTunes can't read my iPod Classic

### ISSUE THREAD

**Issue by zzzuppp** [cormiertyshawn895/Retroactive#108]
cormiertyshawn895 - I had a good look over the thread 'iTunes no longer reading contents of iPod #91' I also have iPod recognition problems. Tried all three versions of iTunes on Retroactive, but the newer ones both said 'can't read this iPod' (it's a Classic, and it reads on my old - High Sierra - Mac in iTunes 12.8.2.3) and it didn't show up on the oldest one at all when connected. So I redownloaded the newest version, and tried the tips in the other thread, downloading the three packages. Two of the pkgs opened and installed, but the iTunes access one says 'This package is incompatible with this version of OS' - 10.15.5 in my case, and won't install. So I tried reinstalling the same version of iTunes once more, as in your last tip about reinstalling the iTunes version already installed once more, but I didn't get the 'One more thing' screenshot...don't know why not, and when I looked at Security/Privacy, iTunes wasn't listed there. It's driving me insane - I have over 10,000 music files on the Classic which I need for my work.
Any more tips? Thanks in advance

**Comment by zzzuppp** [comment]
Sorry! solved it myself by dragging iTunes to Security and Privacy.- and it's working, great! HOWEVER I have another problem - when I installed iTunes from Retroactive  I got the question 'do you want to start a fresh Library' and foolishly/unknowingly clicked 'yes', so all my files that were in iTunes (not the iPod) disappeared. Is there a way I can restore that Library somehow without clicking on dozens of files in my Music folder? Thanks

**Comment by zzzuppp** [comment]
OK solved it by dragging iTunes to Security pane/Full Access...
So my iPod is recognised in iTunes (12.9.5.5) in Catalina. I can play/delete songs, see Playlists.So far so good. Thanks for those tips!

**BUT** I have more problems. 

 I can't import any other songs from my 'Music' library in iTunes Music into iTunes - either by double-clicking or dragging. I get the message 'Attempting to copy to the disk " " fails. The required file cannot be found.'

Meanwhile all the same files **can** be opened in Catalina's 'Music' App - but my iPod won't work there. 
In other words I can't add any new files to the iPod without some way of opening new files in iTunes. I've tried with .aiff, .mp3 etc, none will open from the Music Library.

Any suggestions welcome!

**Comment by kasssie** [comment]
For this issue, you could find and try another way to fix it. Here you could try to move your music to iPod to see whether it can work. You could refer to the post from https://www.drmare.com/apple-music/sync-apple-music-to-ipod.html

**Comment by cormiertyshawn895** [comment]
Retroactive has been discontinued, and does not support macOS Sequoia or later. You should transition from Retroactive to a wide range of supported apps, many of which are built into macOS or free to download. [Learn how to transition from Retroactive to supported apps](https://github.com/cormiertyshawn895/Retroactive/blob/master/TRANSITION.md).
